core/modules/field_group/css/multipage/multipage.css

```diff
@@ -0,0 +1,128 @@
+.multipage-controls-list #edit-actions {
+  float: left; /* LTR */
+}
+
+.multipage-button {
+  margin-bottom: 1em;
+  margin-top: 0;
+  float: left; /* LTR */
+  line-height: 36px;
+}
+
+.multipage-button a {
+  padding-top: 10px;
+}
+
+.multipage-counter {
+  float: right; /* LTR */
+  margin-right: 5px; /* LTR */
+  height: 0;
+  position: relative;
+  top: 1.8em;
+  line-height: 30px;
+  font: 12px arial,sans-serif;
+  font-weight: bold;
+  color:#666;
+}
+
+a.multipage-link-previous {
+  font: 12px arial,sans-serif;
+  font-weight: bold;
+  color:#666;
+  -webkit-transition: color 218ms;
+  -moz-transition: color 218ms;
+  -o-transition: color 218ms;
+  transition: color 218ms;
+}
+
+a.multipage-link-previous:hover {
+  text-decoration:none;
+  color: #333;
+}
+
+.multipage-controls-list input.form-submit {
+  background:none;
+  border: none;
+  border-radius: 2px;
+  -moz-border-radius: 2px;
+  -webkit-border-radius: 2px;
+  border: 1px solid rgba(0, 0, 0, 0.1);
+  font: 12px arial,sans-serif;
+  font-weight: bold;
+  color: #666;
+  text-shadow: 0 1px 0 white;
+  padding: 7px 12px;
+  background: -webkit-gradient(linear,0% 40%,0% 70%,from(whiteSmoke),to(#F1F1F1));
+  -o-transition: border-top-color 0.218s,border-right-color 0.218s,border-bottom-color 0.218s,border-left-color .218s;
```

Comment:
Not 100% sure re coding standards and spacing after commas here 🤔 ...do spaces break things?